tests/manual/a11ychecker.md

```diff
@@ -13,4 +13,5 @@ Some hints:
 - Instead of using "Quick fix", click on the issue source element to edit it manually. A popup should appear in the right-bottom
 corner of the window.
 - Ignore an issue, close the Accessibility Checker dialog and run AC again to find, if the issue is still ignored.
-- Ensure that `esc` key closes the Accessiblity Checker balloon and puts selection on the issue source element.
\ No newline at end of file
+- Ensure that `esc` key closes the Accessiblity Checker balloon and puts selection on the issue source element.
+- Open Accessiblity Checker, press `Shift+Tab` twice. The close button should be focused. Ensure that in this state pressing the `space` key closes Accessiblity Checker.
```

Comment:
Please, put the last TC "- Open Accessiblity Checker, press `Shift+Tab` twice. The close button should be focused. Ensure that in this state pressing the `space` key closes Accessiblity Checker." into a dedicated manual test case.

Create a `.md` file with `tc` tag, and target AC version (1.0.1).